js/screenSnippet/index.js

```diff
@@ -61,6 +63,17 @@ class ScreenSnippet {
                     captureUtil = path.join(execPath, 'ScreenSnippet.exe');
                 }
 
+                // Method to verify and disable always on top property
+                // as an issue with the ScreenSnippet.exe not being on top
+                // of the electron wrapper
+                const windows = electron.BrowserWindow.getAllWindows();
```

Comment:
this looks like it is only is changing isAlwaysOnTop value for first windows (which i would assume is main window) but have you tried this for the pop-outs.  i would assume same issue will happen there.

Reply:
I'm just reading the value here. If `alwaysOnTop` is enabled i'm using the `eventEmitter` to `setAlwaysOnTop` for all the window in `windowMgr.js:501`

Reply:
yes but main question: what happens when always on top is enable and you take screensnippet from a pop-out.  does the issue happen there as well?  or this only an issue for the main window?

Reply:
Yes this issue happens both on pop-out and main window.

Reply:
ok i see, lgtm.